src/reducers/dummyData.js

```diff
@@ -0,0 +1,13 @@
+import {Musician, Band} from './profileClasses';
+
+const dummyInitialState = {
+  musicians: {
+    14343: new Musician('jiri', 'yadiya'),
+
```

Comment:
Misschien kunnen we deze `musicians` en `bands` beter combineren tot een enkele `profiles`